### PR TITLE
hopefully this makes W a little better

### PR DESCRIPTION
**Pull request by Lyxal**

### CODE REVIEW

**Review by razetime — APPROVED** [review]

File: vyxal/elements.py [hunk 1]
```diff
@@ -4536,10 +4536,9 @@ def zfiller(lhs, rhs, ctx):
     "U": process_element(uniquify, 1),
     "V": process_element(replace, 3),
     "W": (
-        "temp = stack[::]\n"
-        "for item in stack:\n"
-        "    stack.pop()\n"
-        "stack.append(temp)",
+        "temp = list(deep_copy(stack))\n"
+        "pop(stack, len(stack), ctx)\n"
+        "stack.append(temp); print(stack)",
```
Comment (pxeger) [inline vyxal/elements.py:4541]: Way simpler:
```suggestion
        "stack[:] = [deep_copy(stack)]"
```

**Review by pxeger — APPROVED** [review]
I mean if it works, then fine, but this does not need to be this complicated

**Review by cgccuser — APPROVED** [review]

File: vyxal/elements.py [hunk 2]
```diff
@@ -4536,10 +4536,7 @@ def zfiller(lhs, rhs, ctx):
     "U": process_element(uniquify, 1),
     "V": process_element(replace, 3),
     "W": (
-        "temp = stack[::]\n"
-        "for item in stack:\n"
-        "    stack.pop()\n"
-        "stack.append(temp)",
+        "stack[:] = [deep_copy(stack)]"
```
Comment (cgccuser) [inline vyxal/elements.py:4539]:
```suggestion
        "stack[:] = [deep_copy(stack)]",
```

**Review by cgccuser — CHANGES_REQUESTED** [review]

**Comment by cgccuser** [comment]
Boy this PR turned into a mess...

**Review by cgccuser — APPROVED** [review]

**Comment by cgccuser** [comment]
Okay imma just check it out locally and use `stack[:] = [stack[:]]` or something

File: vyxal/elements.py [hunk 3]
```diff
@@ -4536,10 +4536,7 @@ def zfiller(lhs, rhs, ctx):
     "U": process_element(uniquify, 1),
     "V": process_element(replace, 3),
     "W": (
-        "temp = stack[::]\n"
-        "for item in stack:\n"
-        "    stack.pop()\n"
-        "stack.append(temp)",
+        "stack[:] = [deep_copy(stack)]",
```
Comment (cgccuser) [inline vyxal/elements.py:4539]:
```suggestion
        "temp = list(deep_copy(stack))\n"
        "pop(stack, len(stack), ctx)\n"
        "stack.append(temp); print(stack)",
```

**Pull request merged**